src/test/java/it/gov/pagopa/pu/debtpositions/service/InstallmentServiceImplTest.java

```diff
@@ -71,6 +74,7 @@ class InstallmentServiceImplTest {
   void setUp() {
     installmentService = new InstallmentServiceImpl(
       installmentPIIRepositoryMock,
+      installmentNoPIIRepositoryMock,
```

Comment:
let's add this also to verifyNoMoreInteractions method!
this should fail the actual tests, because you are not testing the modified code